src/Fmt_ast.ml

```diff
@@ -2439,11 +2437,15 @@ and fmt_class_expr c ?eol ?(box = true) ({ast= exp; _} as xexp) =
   | Pcl_fun _ ->
       let xargs, xbody = Sugar.cl_fun c.cmts xexp in
       let stmt_loc = Sugar.args_location xargs in
+      let args_box =
```

Comment:
This is duplicated every time. I suggest returning the `args_box` from `wrap_fun_decl_args`.

Reply:
Done.